app/controllers/api/v2/discovered_hosts_controller.rb

```diff
@@ -197,9 +197,7 @@ def reboot_all
         if error_message
           render_error :custom_error,
```

Comment:
After discussion with @tstrachota we should create our own discovery RABL which can extend using `extends` keywoard from `custom_error` and then you can use this here on this line to render it. No core change would be required then.

Reply:
awesome :) will do that ! closing -  theforeman/foreman#4875 ?